viz.yaml

```diff
@@ -91,12 +98,26 @@ publish:
     id: text-section
     template: printall
     publisher: section
-    context: 
-      text: 
+    context:
+      text:
         This is a text section in the viz.yaml.  Since it is only
         text, it doesn't need be referenced anywhere outside of the publish
         section.  Text wrapping in the viz should be handled automatically.
-        Follow YAML conventions to change formatting.  
+        Follow YAML conventions to change formatting.
+  -
+    id: markdown-section1
+    template: printall
+    publisher: section
+    depends: markdown-text
+    context:
+      text: markdown-text.example-markdown
+  -
+    id: markdown-section2
+    template: printall
+    publisher: section
+    depends: markdown-text
+    context:
+      text: markdown-text.markdown-table
```

Comment:
could you add/edit an example to look more like what @jread-usgs is sharing in slack right now? 
instead of
```
      line4: "Precipitation: National Weather Service Environmental Modeling Center National Center for Environmental Prediction "
      emcText: "Stage IV precipitation analysis"
      emcLink: "http://www.emc.ncep.noaa.gov/mmb/ylin/pcpanl/stage4/"
      line5: ", aggregated according to "
```
we do:
```
line4: "Precipitation: National Weather Service Environmental Modeling Center National Center for Environmental Prediction (Stage IV precipitation analysis)[http://www.emc.ncep.noaa.gov/mmb/ylin/pcpanl/stage4/], aggregated according to "
```
?

Reply:
Yeah, I have to take another look at this.